src/status_im/contexts/wallet/add_account/create_account/events.cljs

```diff
@@ -30,6 +31,13 @@
 
 (rf/reg-event-fx :wallet/store-secret-phrase store-secret-phrase)
 
+(defn seed-phrase-entered
+  [{:keys [db]} [{:keys [seed-phrase]}]]
+  {:db (assoc-in db [:wallet :ui :create-account :secret-phrase] seed-phrase)
```

Comment:
why do we refer to it as `secret-phrase` here by `seed-phrase` elsewhere? should it not be consitent?